### PR TITLE
Prepare a 1.7.0 release

### PR DESCRIPTION
**Pull request by bmerry**

### CODE REVIEW

**Comment by bmerry** [comment]
@ludwigschwardt hold off on this if you haven't started yet - it will need an update after #319 is merged.

**Comment by bmerry** [comment]
It seems like there is at least one more behavioural corner case that hypothesis is picking up. I'll have to get back to this next Monday.

**Comment by bmerry** [comment]
@ludwigschwardt I think I've finally got all the tests to pass, so please take a look now.

File: fakeredis/_server.py
Comment (ludwigschwardt) [inline fakeredis/_server.py:2168]: Do you test this with multiple args somewhere?

File: test/test_fakeredis.py
Comment (ludwigschwardt) [inline test/test_fakeredis.py:730]: I would repeat the `# Overflows long long in ms` comment (in case the other test is deleted some day).

File: test/test_fakeredis.py
Comment (ludwigschwardt) [inline test/test_fakeredis.py:735]: Does this also overflow? Why a different value?

File: test/test_fakeredis.py
Comment (ludwigschwardt) [inline test/test_fakeredis.py:713]: What is the difference between SETEX and SET_EX?

**Review by ludwigschwardt — CHANGES_REQUESTED** [review]
Minor suggestions - take it or leave it :-)

File: test/test_fakeredis.py
Comment (bmerry) [reply test/test_fakeredis.py:713]: setex corresponds to the redis SETEX command. set_ex corresponds to the redis SET command with the EX option.

File: fakeredis/_server.py
Comment (bmerry) [reply fakeredis/_server.py:2168]: The Hypothesis-based fuzz tester does, which is what prompted this. I haven't bothered writing a targeted test since it's not something I expect any code in the wild to actually do.

**Review by ludwigschwardt — APPROVED** [review]
👍🏾

**Comment by bmerry** [comment]
I'm running into yet another test failure, but I'll raise a new PR for it and merge this so long.

**Pull request merged**
